.circleci/config.yml

```diff
@@ -3,6 +3,7 @@ version: 2.1
 orbs:
   docker: circleci/docker@1.0.1
   slack: circleci/slack@3.4.1
+  puppeteer: threetreeslight/puppeteer@0.1.2
```

Comment:
This gets around this error: https://app.circleci.com/pipelines/github/statechannels/monorepo/6062/workflows/28924b87-cb72-497c-930c-a5eacd549a11/jobs/24564

I'm not sure if there is a better solution.